js/reference.js

```diff
@@ -6595,3 +6618,126 @@
             }
         });
     }
+
+    /**
+     *
+     * @param {ERMrest.Reference} reference reference for the association table
+     * @param {Object} prefillObject computed prefill object after extracting the query param and fetching the data from cookie storage
+     * @param {Object} prefillAnnotation
+     */
+    function Prefill (reference, prefillObject, prefillAnnotation) {
+        verify(prefillObject, "'prefillObject' must be defined");
+
+        var self = this;
+        this._reference = reference;
+        this._prefillObject = prefillObject;
```

Comment:
One change I mentioned in Chaise code (I think) was that maybe I should move `prefillObject` entirely from chaise and have it as part of the reference. If I do that, then there should be a `get prefillObject()` function in the prototype of `Prefill`